### ISSUE TITLE
Integrate with magit for better visibility

### ISSUE THREAD

**Issue by larebsyed** [ag91/code-compass#12]
We can add  a section in magit buffer to show the files coupled with currently edited files it git

**Comment by ag91** [comment]
How would you use that? As a reminder?

**Comment by larebsyed** [comment]
Yes before commit one would be able to see all the coupled file related to currently uncommited files. This should integrate the feature into already active workflow

**Comment by larebsyed** [comment]
@ag91 What do you think about this feature?

**Comment by ag91** [comment]
It is interesting! I was planning to use magit's status buffer
anyway at some point. If you want to give a try at implementing this, I
would welcome a PR. (Otherwise you will have to wait a bit because I am
working also on some other projects at the moment...)
